ams_flutter/lib/features/onboarding/presentation/pages/signup.dart

```diff
@@ -0,0 +1,224 @@
+import 'package:flutter/material.dart';
+
+class signup_page extends StatefulWidget {
+  const signup_page({super.key});
+
+  @override
+  State<signup_page> createState() => _signup_pageState();
+}
+
+class _signup_pageState extends State<signup_page> {
+  bool _rememberMe = false;
+  bool _obscureText = true;
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+        debugShowCheckedModeBanner: false,
+        home: Scaffold(
+          appBar: AppBar(
+            title: Row(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                Image.asset(
+                  'Assets/images/img.png',
```

Comment:
The variable has already been setup in `core/constants` directory. Import that variable here